.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "max-len": "off",
+      "no-use-before-define": "off",
+      "import/no-cycle": "off"
```

Comment:
- [ ] This file is different from the one provided by Microverse [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). We should not make changes in this file